src/assets/css/global/_main.scss

```diff
@@ -618,13 +618,31 @@ km-cni-version {
 }
 
 .km-quota-widget {
-  .km-quota-item {
+  km-property {
+    width: 100%;
+
     .container {
-      margin: 2px 5px;
+      width: 100%;
+
+      &:not(last-child) {
```

Comment:
This selector has no effect apparently. It applies this margin to all `km-property` elements.